test/controllers/admin_users_controller_test.rb

```diff
@@ -0,0 +1,53 @@
+require "test_helper"
```

Comment:
IMHO, rather than minitest, rspec is preferred here bhai. Please take prior approval from your mentor if you want to use minitest

Reply:
I will use Rspec later at the project.After finishing the controller I will test everything using Rspec

Reply:
same goes for i18n bhai, you may need to convert hardcoded string to i18n

Reply:
i Created a different branch where i only worked on i18n . I changed every hardcoded string with i18n